### PR TITLE
Feature/327 3 implement additional plots from functional requirements

### PR DESCRIPTION
**Pull request by nikgraf**
https://trello.com/c/ZNhXlABl/327-3-implement-additional-plots-from-functional-requirements

### CODE REVIEW

**Comment by nikgraf** [comment]
@Ashalynd all export should work now:

* data for stacked charts
* data for cost coverage

I changed the colors for the stacked area graphs as well.

File: client/source/js/modules/analysis/optimization-ctrl.js [hunk 1]
```diff
@@ -120,15 +113,6 @@ define(['./module', 'angular', 'd3'], function (module, angular, d3) {
     };
 
     /*
-    * Returns an array containing arrays with [x, y] for d3 line data.
```
Comment (nikgraf) [inline client/source/js/modules/analysis/optimization-ctrl.js]: We simply use _.zip now

File: client/source/js/modules/analysis/optimization-ctrl.js [hunk 2]
```diff
@@ -120,15 +113,6 @@ define(['./module', 'angular', 'd3'], function (module, angular, d3) {
...
-    */
-    var generateLineData = function(xData, yData) {
```
Comment (nikgraf) [inline client/source/js/modules/analysis/optimization-ctrl.js]: Replaced it with _.zip function as it provides exactly the same result

**Comment by Ashalynd** [comment]
I have found one issue: stacked charts could become misaligned. I fixed it by forcing all grid cells to be higher in we have at least one stacked chart. Not the best solution, but better than gaps.

**Comment by nikgraf** [comment]
@Ashalynd ahh, that might happen in bigger screens. sorry, forget about this case

**Pull request merged**